### PR TITLE
Add possibility to allow list  of strings in configuration

### PR DESCRIPTION
**Pull request by anusudarsan**
This will allow the possibility to add list of strings in the
configuration. Eg : the yaml will look like:

key:
 - value1
 - value2
 - etc

Task: Issue #50, SWARM-
Testing: gradle test

### CODE REVIEW

File: tempto-core/src/main/java/com/teradata/tempto/configuration/Configuration.java [hunk 1]
```diff
@@ -75,4 +76,6 @@
     Configuration getSubconfiguration(String keyPrefix);
 
     Map<String, Object> asMap();
+
+    List<String> getStringList(String key);
```
Comment (losipiuk) [inline tempto-core/src/main/java/com/teradata/tempto/configuration/Configuration.java]: Move up to section with the rest of get* methods.
Consider adding getStringListMandatory methods for completeness.

**Comment by losipiuk** [comment]
looks good

**Comment by anusudarsan** [comment]
@losipiuk : addressed comment

**Comment by kokosing** [comment]
nice, lgtm

**Pull request closed without merging**